src/components/Tile.js

```diff
@@ -0,0 +1,42 @@
+import { useGameContext } from '../context/GameContext.js';
+import './Tile.css';
+// import horse from '../../public/horse.png';
+// import snake from '../../public/snake.png';
+export default function Tile({ tileContent, index }) {
+  const { player, setPlayer, board, active, setBoard, setMessage } = useGameContext();
+
+  function handleClick() {
```

Comment:
you could've placed handleClick in GameContext as well but I think it makes sense to live inside this component